src/cpr_sdk/search_adaptors.py

```diff
@@ -90,6 +90,8 @@ def search(self, parameters: SearchParameters) -> SearchResponse:
                 raise e
         query_time_end = time.time()
 
+        # TODO: Confirmed that here we have geography but no geographies in the
```

Comment:
Items should also be on the passage that are not on the passage so we can return that context - but that doesn't explain why vespa is returning nothing. I think we can fix this one with a schema change and redeploy